app/models.py

```diff
@@ -853,21 +864,31 @@ def website_send_to(self):
 
     def new_addr(self):
         """
-        Replace original email by reply_email. 2 possible formats:
+        Replace original email by reply_email. Possible formats:
         - first@example.com by SimpleLogin <reply_email> OR
-        - First Last - first at example.com <reply_email>
+        - First Last - first at example.com <reply_email> OR
+        - First Last - first@example.com <reply_email>
         And return new address with RFC 2047 format
 
         `new_email` is a special reply address
         """
         user = self.user
-        if user and user.use_via_format_for_sender:
+        if (
+            not user
+            or not SenderFormatEnum.has_value(user.sender_format)
+            or user.sender_format == SenderFormatEnum.VIA.value
+        ):
             new_name = f"{self.website_email} via SimpleLogin"
-        else:
+        elif user.sender_format == SenderFormatEnum.AT.value:
             name = self.name or ""
             new_name = (
                 name + (" - " if name else "") + self.website_email.replace("@", " at ")
             ).strip()
+        elif user.sender_format == SenderFormatEnum.C_AT.value:
+            name = self.name or ""
+            new_name = (
+                name + (" - " if name else "") + self.website_email.replace("@", "@")
```

Comment:
the `replace` here can be removed I think :)

Reply:
Do we want to try the different at symbol? https://trello.com/c/30iiOVXS
https://unicode-table.com/en/search/?q=at

Reply:
Yeah we definitely can! But I think we need to make sure user understands that these are 2 different characters on the Settings page to avoid surprises.

Reply:
So the reasoning is that some email clients do not display the symbol correctly? Do you have an example so I can do some tests?

Reply:
Ah the main reason is sometimes people can try to copy/paste the sender email address that would not work because the character is not the usual `@`.